### ISSUE TITLE
Wi-Fi checkbox shows as "enabled" regardless of actual Wi-Fi state

### ISSUE THREAD

**Issue by gbyl** [koreader/koreader#11257]
* KOReader version: 2023.10-70
* Device: reMarkable 1

#### Issue
Unable to uncheck UI checkbox for "Wi-Fi connection"

#### Steps to reproduce
1. Start KOReader
2. Navigate to Settings > Network
3. "Wi-Fi connection" checkbox is enabled
4. Tap on it to trigger the prompt "Wi-Fi is enabled, but you're currently not connected to a network."
5. Choose "Turn Wi-Fi off"
6. "Wi-Fi connection" checkbox remains enabled

Above steps also tested on a fresh install. Wi-Fi state double-checked as off with `curl` at every step.

**Comment by NiLuJe** [comment]
I have absolutely no clue how Wi-Fi works on rM, so you or someone with access to a device will have to poke around a bit.

The checkbox is tied to `isWifiOn`, which assume true as long as the interface *exists* (which usually implies the driver is loaded).

It's plausible the system no longer (or never did) ever unload the driver, so you'll need to device a better way to handle that in a very efficient manner.

Right now, it's effectively checking for this:

https://github.com/koreader/koreader/blob/e7ee900cbd0fe189ee263080dbfe9130b071726b/frontend/ui/network/manager.lua#L187-L191

**Comment by gbyl** [comment]
>It's plausible the system no longer (or never did) unloads the driver, so you'll need to devise a better way to handle that in a very efficient manner.

Thank you so much NiLuje! This was the key to figuring it out. As far as I can tell the stock rM1 kernel (Linux reMarkable 5.4.70-v1.2.5) does not set `CONFIG_MODULE_UNLOAD`.

```
$ lsmod
Module                  Size  Used by
brcmfmac              237568  -2
cfg80211              319488  -2
brcmutil               16384  -2

$ rmmod -f brcmfmac
rmmod: ERROR: could not remove module brcmfmac: Function not implemented
```

So, that would mean this line never succeeds:

https://github.com/koreader/koreader/blob/9ba66ac382ab8778978d075d0de15b73617de49b/platform/remarkable/disable-wifi.sh#L13-L14

The only workaround I found was to prevent the kernel module from loading in the first place in `/etc/modprobe.d/blacklist.conf`

```
blacklist brcmfmac
blacklist brcmutil
blacklist cfg80211
```

And then I had to disable the DHCP service to avoid a conflict/timeout on startup. So, I also added `systemctl start dhcpcd` and `systemctl stop dhcpcd` to the enable/disable Wi-Fi scripts. It works, but it's probably out of scope for a KOReader PR.
